packages/desktop-client/src/components/settings/Encryption.tsx

```diff
@@ -79,13 +84,16 @@ export function EncryptionSettings() {
           to="https://actualbudget.org/docs/getting-started/sync/#end-to-end-encryption"
           linkColor="purple"
         >
-          Learn more…
+          {t('Learn more…')}
         </Link>
       </Text>
     </Setting>
   ) : (
-    <Setting primaryAction={<Button isDisabled>Enable encryption…</Button>}>
+    <Setting
+      primaryAction={<Button isDisabled>{t('Enable encryption…')}</Button>}
+    >
       <Text>
+        <strong>{t('End-to-end encryption')}</strong>{' '}
```

Comment:
This will be why the E2E tests are failing - You've duplicated the "End-to-end encryption" text. The other text needs translated too